.travis.yml

```diff
@@ -71,6 +71,7 @@ script:
          python pymathics/test.py -t &&
          cd pymathics/doc/tex &&  make latex;
       else
+         python test.py -o
```

Comment:
I think this line is redundant. In other words, we might be running the tests twice. I'll investigate this.

Reply:
Yes, this line made it so that the tests were run twice. That wouldn't be fun, the tests already take up to 20 minutes to finish.